tests/js/spec/views/performance/landing/widgets/widgetContainer.spec.jsx

```diff
@@ -1146,90 +1146,87 @@ describe('Performance > Widgets > WidgetContainer', function () {
         })
       );
     });
-  });
-
-  it('TPM - Single Area Widget - metrics based', async function () {
-    metricsMock = MockApiClient.addMockResponse({
-      method: 'GET',
-      url: `/organizations/org-slug/metrics/data/`,
-      body: TestStubs.SingleFieldArea({field: 'count(transaction.duration)'}),
-      match: [(...args) => !issuesPredicate(...args)],
-    });
 
-    const metricsMockPreviousData = MockApiClient.addMockResponse({
-      method: 'GET',
-      url: `/organizations/org-slug/metrics/data/`,
-      body: TestStubs.SingleFieldArea({
-        field: 'count(transaction.duration)',
-        previousData: true,
-      }),
-      match: [
-        (...args) => {
-          return (
-            !issuesPredicate(...args) &&
-            args[1].query.statsPeriodStart &&
-            args[1].query.statsPeriodEnd
-          );
-        },
-      ],
-    });
+    it('Failure Rate', async function () {
```

Comment:
```suggestion
    it('Failure Rate Widget - metrics based', async function () {
```

Reply:
Failure Rate is now inside of `describe('Single Area Widget - metrics based', function () {`